src/components/DevfilePage/DevfilePageProjects/DevfilePageProjects.tsx

```diff
@@ -79,15 +79,18 @@ export const DevfilePageProjects: React.FC<DevfilePageProjectsProps> = ({
       const region = getUserRegion(router.locale);
       const { publicRuntimeConfig } = getConfig();
 
-      analytics.track({
-        userId: publicRuntimeConfig.segmentUserId,
-        event: 'Starter Project Downloaded',
-        properties: {
+      analytics.track(
```

Comment:
we can do this later, but probably a good idea to track after the call to `await download(project);` has finished. If there is an exception in the download function and a user spam clicks download, this will just spam analytics to thinking a user really clicked on this project download many times 🤔